content/millennium/r4/medications/medication-request.md

```diff
@@ -59,6 +59,20 @@ The following fields are returned if valued:
 
 <%= terminology_table(:medication_request_contained_medication, :r4) %>
 
+## Extensions
+
+* [NLLPrescriptionFormat](http://electronichealth.se/fhir/StructureDefinition/NLLPrescriptionFormat)
+* [NLLRegistrationBasis](http://electronichealth.se/fhir/StructureDefinition/NLLRegistrationBasis)
+* [NLLDosePackaging](http://electronichealth.se/fhir/StructureDefinition/NLLDosePackaging)
+
+All URLs for hardcoded extensions are defined as `http://electronichealth.se/fhir/StructureDefinition/{id}`
```

Comment:
I don't think we want 'hardcoded' listed here. Maybe it should be something like "All URLs for Swedish extensions are defined as `http://electronichealth.se/fhir/StructureDefinition/{id}`

Reply:
Fix for this:
https://github.com/cerner/fhir.cerner.com/commit/a92347cb1bc1b50ba45b882417bda299dfc99c6b